docs/src/main/sphinx/security/oauth2.rst

```diff
@@ -114,6 +114,39 @@ The following configuration properties are available:
    * - ``http-server.authentication.oauth2.groups-field``
      - The field of the access token used for Trino groups. The corresponding claim value must be an array.
 
+Refresh Tokens
+^^^^^^^^^^^^^^
+
+In order to start using refresh tokens flows, the following property must be enabled it in the coordinator configuration.
+
+.. code-block:: properties
+
+    http-server.authentication.oauth2.refresh-tokens=true
+
+``offline`` scope should also be specified as a part of oauth2 scopes
```

Comment:
I would rather go with something like: `additional scopes for offline access might be required, depending on your IdP configuration`